<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiTransactionList</class>
 <widget class="QWidget" name="GuiTransactionList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabelTitle">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>&lt;font size=&quot;+2&quot;&gt;&lt;u&gt;Turnovers:&lt;/u&gt;&lt;/font&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <layout class="QVBoxLayout">
       <item>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabelAccounts">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Account:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="textLabelAccount">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabelDate">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Period:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxDate">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <item>
            <property name="text">
             <string>all</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>current year</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>last year</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>current month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>last month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>current quarter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>last quarter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>from to</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frameFromTo">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>From:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEditFrom">
              <property name="minimumDate">
               <date>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>dd.MM.yyyy</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>To:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEditTo">
              <property name="minimumDate">
               <date>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="displayFormat">
               <string>dd.MM.yyyy</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Purpose</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bank Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Account Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Text</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Booking Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valuta Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Store Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Primanota</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Customer Reference</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame3">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="textLabelIn">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Incoming Payments</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditIncome"/>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="textLabelOut">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Outgoing Payments</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditOutGo"/>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>81</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPrintList">
       <property name="text">
        <string>Print &amp;List</string>
       </property>
       <property name="shortcut">
        <string>Alt+L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExportList">
       <property name="text">
        <string>&amp;Export List</string>
       </property>
       <property name="shortcut">
        <string>Alt+E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreateTransferTemplates">
       <property name="text">
        <string>Create &amp;Transfer Templates</string>
       </property>
       <property name="shortcut">
        <string>Alt+T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpdateData">
       <property name="text">
        <string>&amp;Update Data</string>
       </property>
       <property name="shortcut">
        <string>Alt+U</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
